moqt-server/src/modules/message_handlers/control_message/handlers/subscribe_ok_handler.rs

```diff
@@ -48,19 +48,19 @@ pub(crate) async fn subscribe_ok_handler(
                     subscribe_ok_message.largest_object_id(),
                     subscribe_ok_message.subscribe_parameters().clone(),
                 );
-                let relaying_subscribe_ok_message: Box<dyn MOQTPayload> =
+                let forwarding_subscribe_ok_message: Box<dyn MOQTPayload> =
```

Comment:
I think `subscribe_ok_message` is enough to understand.

Reply:
I have fixed.